docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: "3.3"
 services:
   iroha:
-    image: iroha1/iroha:iroha2-dev
+    image: hyperledger/iroha2:iroha2-dev
```

Comment:
May be we can make tags like `hyperledger/iroha2:dev` `hyperledger/iroha2:cli-dev` `hyperledger/iroha2:crypto-cli-dev`

Reply:
I agree. Now that the repository is dedicated to i2, the tags are natural